cmd/store/import.go

```diff
@@ -44,8 +44,13 @@ func importStore(
 ) (*CreateStoreAndModelResponse, error) {
 	var err error
 	var response *CreateStoreAndModelResponse //nolint:wsl
-	if storeID == "" {                        //nolint:wsl
-		createStoreAndModelResponse, err := CreateStoreWithModel(clientConfig, storeData.Name, storeData.Model, format)
+	if storeID == "" {                        //nolint:wsl,nestif
+		storeDataName := storeData.Name
+		if storeDataName == "" {
+			storeDataName = storeData.Model
```

Comment:
I think here we potentially want to use [path.Base](https://pkg.go.dev/path#Base) to get the filename, and then use [strings.TrimSuffix](https://pkg.go.dev/strings#TrimSuffix) to remove `fga.yaml` from the filename which should leave us with the store name to use

Reply:
@ewanharris I have not taken the fileName as an argument in importStore function and extracted the store name part.Am I correct in this approach?

Reply:
@NeerajNagure I think it's nearly there. Assuming I'm running `fga store import --file /Users/ewan/my_store.fga.yaml` then as we're currently doing `strings.TrimSuffix(fileName, ".fga.yaml")` we will end up with a store name that includes path (i.e. `/Users/ewan/my_store`), whereas we want just `my_store`,

Doing `storeDataName = strings.TrimSuffix(path.Base(fileName), ".fga.yaml")` should allow us to retrieve the filename (`my_store.fga.yaml`)  and then trim `".fga.yaml"` from it to get `my_store`